### ISSUE TITLE
Errors with script v2.3.0

### ISSUE THREAD

**Issue by poshaughnessy** [atatus/atatus-js#3]
Hi there,

We're seeing lots of errors cropping up sometimes with the version 2.3.0 of the script we have installed in a Cordova app.

It appears to fail to load resources and has JavaScript errors:

<img width="1427" alt="screen shot 2016-04-07 at 12 13 36" src="https://cloud.githubusercontent.com/assets/151207/14349451/0c6ab566-fcbb-11e5-89d7-c483aae37b40.png">

Expanded view of the JS errors:

<img width="1063" alt="screen shot 2016-04-07 at 12 27 15" src="https://cloud.githubusercontent.com/assets/151207/14349621/27444112-fcbc-11e5-998a-940fe7f1b856.png">

The network tab also shows it hanging and failing on these requests, which seems to delay our other app requests.

<img width="1213" alt="screen shot 2016-04-07 at 12 24 23" src="https://cloud.githubusercontent.com/assets/151207/14349537/b2e39afc-fcbb-11e5-8a5c-f9d30ffdca1a.png">

This appears to be causing problems with a user trial that is in progress, so I would really appreciate any help you could give us! Thanks.

p.s. The device was an HTC Desire 610 running Android 4.4.2

**Comment by fizerkhan** [comment]
@poshaughnessy Could you give me the following details:
1. Are you testing it directly on the device or from Chrome
2. Is it happening every time or is it sporadically reproducible

The error seems to be that our backend is not reachable from the device. I noticed that you are receiving errors now in your project. Are you still seeing the issue?

**Comment by poshaughnessy** [comment]
Hi. Yesterday we upgraded the script to v2.3.5 and that appears to have resolved the issue for us.

I was testing directly on the device and debugging remotely with Chrome.

The initial error appeared to happen consistently time with v2.3.0. NB. It wasn't on load, but once we tried to log in on the first page. The slow / hanging requests seemed to be more sporadic - that didn't happen every time.
